docs/releaseNotes.yml

```diff
@@ -32,6 +32,15 @@
 
 changelog: https://github.com/emissary-ingress/emissary/blob/$branch$/CHANGELOG.md
 items:
+  - version: 3.1.0
+    date: 'TBD'
+    notes:
+      - title: Add support for OpenAPI 2 contracts
+        type: feature
+        body: >-
+          The agent is now able to parse api contracts using swagger 2, and to convert them to OpenAPI 3, making them
+          available for use in the dev portal.
```

Comment:
To be clear does this make them available in the local devportal that ships with edgestack or only surfaces up in the cloud? If so it might make sense to be clear that it is the Ambassador Cloud DevPortal. If available, then ignore me :).

Reply:
It will work with the local dev portal as-well, because the UI lib used support both open api 2 & 3.

The cloud version will only expose version 3 though through the conversion following one of @alexgervais comment (he was feeling like it was enough to just convert).